CHANGELOG.md

```diff
@@ -4,10 +4,22 @@ Changelog for NeoFS Archive
 
 ## [Unreleased]
 
+## [0.0.2] - 2025-02-27
+
+### Fixed
+
+- Reverse range handling (#5)
+- Allow to input valid border in case of failed getblockcount request (#7)
+
+### Added
+
+- Stop & pause buttons with disable caching (#8)
+
 ## [0.0.1] - 2025-02-17
 
 First public release.
 
 
 [0.0.1]: https://github.com/nspcc-dev/archive-fs-neo-org/tree/v0.0.1
-[Unreleased]: https://github.com/nspcc-dev/archive-fs-neo-org/compare/v0.0.1...master
+[0.0.2]: https://github.com/nspcc-dev/archive-fs-neo-org/tree/v0.0.2
```

Comment:
Make it a compare link, like `https://github.com/nspcc-dev/neofs-node/compare/v0.39.1...v0.39.2`, this can't be done for 0.0.1, but it's a better option for subsequent ones.